### ISSUE TITLE
Compass Gems error after updating Ruby

### ISSUE THREAD

**Issue by emoggio** [Compass/compass#1962]
after updating ruby compass doesn't work anymore. every time I do compass watch filename everything is smooth but as soon as I save i get the following error:

`>>> Compass is watching for changes. Press Ctrl-C to Stop.
[Listen warning]:
  Listen will be polling for changes. Learn more at https://github.com/guard/listen#polling-fallback.
modified filename/sass/style.scss
    error filename/sass/style.scss (Line 7 of /Users/username/.rvm/gems/ruby-2.2.0/gems/compass-core-1.0.3/stylesheets/compass/_support.scss: Did not recognize your adapter specification (cannot load such file -- multi_json/adapters/json_gem). forbrowsers')`

how do I solve it?

**\* LOCAL GEMS ***

activesupport (4.2.1)
bigdecimal (1.2.6)
builder (3.2.2)
bundler (1.9.1)
bundler-unload (1.0.2)
chunky_png (1.3.4)
compass (1.0.3)
compass-core (1.0.3)
compass-import-once (1.0.5)
erubis (2.7.0)
executable-hooks (1.3.2)
ffi (1.9.8)
gem-wrappers (1.2.7)
i18n (0.7.0)
io-console (0.4.3)
json (1.8.1)
mail (2.6.3)
memcache-client (1.8.5)
mime-types (2.4.3)
mini_portile (0.6.2)
minitest (5.4.3)
multi_json (1.11.0)
power_assert (0.2.2)
psych (2.0.8)
rack (1.6.0)
rack-mount (0.8.3)
rack-test (0.6.3)
rails-deprecated_sanitizer (1.0.3)
rake (10.4.2)
rb-fsevent (0.9.4)
rb-inotify (0.9.5)
rdoc (4.2.0)
rubygems-bundler (1.4.4)
rvm (1.11.3.9)
sass (3.4.13)
sqlite3 (1.3.10)
sqlite3-ruby (1.3.3)
test-unit (3.0.8)
thor (0.19.1)
thread_safe (0.3.5)
tzinfo (1.2.2)

**Comment by karljakober** [comment]
Having the same error... Did you fix it? I am currently slowly downgrading ruby to find the breaking point.

**Comment by emoggio** [comment]
Nope! I asked a bit around and apparently it happened to a few friends of mine already, they weren't also able to fix it. I uninstalled everything and reinstalled but the problem persist. My guess is that there is some kind of problem between the ruby version and the compass version. 
If you use mac and have a time machine backup from shortly before it happened I would try to restore it, don't know if it overwrites ruby version and compass, might possibly work but is a bit brutal. 
My solution was to uninstall compass and upgrade to gulp. No problem since.

**Comment by emoggio** [comment]
let me know if you find out, I'm really curious now!

**Comment by zyuhel** [comment]
had the same issue. uninstalled compass with all packages such as compass-core, and sass. when reinstalled both. in new terminal session everything works fine now.
